setup.py

```diff
@@ -20,6 +20,11 @@
 from shutil import copy
 
 try:
+    import setuptools
+except ImportError:
+    pass  # setuptools allows for "develop" and entry_points, but it's not essential
```

Comment:
yes, this is enough for setuptools to latch itself to distutils (in the dirty way that it does this), so that `setup()` will know about entry points et. al.